src/main/java/thePackmaster/powers/marisapack/FreeAmplifyPower.java

```diff
@@ -0,0 +1,42 @@
+package thePackmaster.powers.marisapack;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class FreeAmplifyPower extends AbstractPackmasterPower implements CloneablePowerInterface, AmplifyPowerHook {
```

Comment:
While reviewing this it hit me that since this power doesn't expire at the end of the turn, you will always get value from Blazing Star as long as you have cards with Amplify in your deck. It just might be delayed until you draw your next Amplify card.

This also means that the more Amplify cards you have, the more Blazing Stars you want, and the more Blazing Stars you have, the more Amplify cards you want. A virtuous cycle. Card draw would be the third piece of the puzzle, so it might be a good thing that Marisa doesn't have much, means you have to look to other packs to really fuel this kind of play.

Makes me really interested to see how well this works in practice.

Reply:
Yeah, I always saw this as a Flex + delayed energy gain. I was hoping this would be a good incentive to go for Amplify cards.